draft-ietf-tls-tls13.md

```diff
@@ -1517,7 +1517,7 @@ Structure of this message:
            Random random;
            opaque legacy_session_id<0..32>;
            CipherSuite cipher_suites<2..2^16-2>;
-           opaque legacy_compression_methods<1..2^8-1>;
+           opaque legacy_compression_methods[2] = 0x0100;
```

Comment:
legacy_compression_methods as opaque[2] feels more natural than uint16 to me, as it's just hardcoding the encoding of a specific vector; it's not really a two-byte integer or constant.  (Well, it is a two-byte constant, but that's "coincidence".)  For CipherSuite it seems to make sense, though.